src/opera/commands/outputs.py

```diff
@@ -30,15 +31,6 @@ def add_parser(subparsers):
     parser.set_defaults(func=_parser_callback)
 
 
-def format_outputs(outputs, format):
-    if format == "json":
-        return json.dumps(outputs, indent=2)
-    if format == "yaml":
-        return yaml.safe_dump(outputs, default_flow_style=False)
-
-    assert False, "BUG - invalid format"
-
-
```

Comment:
Great! You can now also remove unused `import json` and `import yaml` imports from this file.